src/server.cpp

```diff
@@ -442,15 +445,12 @@ void Server::init()
 	servermap->addEventReceiver(this);
 
 	m_env->loadMeta();
-
-	m_liquid_transform_every = g_settings->getFloat("liquid_update");
-	m_max_chatmessage_length = g_settings->getU16("chat_message_max_size");
-	m_csm_restriction_flags = g_settings->getU64("csm_restriction_flags");
-	m_csm_restriction_noderange = g_settings->getU32("csm_restriction_noderange");
```

Comment:
I'd prefer to keep the settings getters nicely grouped here.

Reply:
okay, as you like

Reply:
i understand, but they are inited outside of the constructor. I will add them grouped in the constructor